CRM/Upgrade/Incremental/php/SixEight.php

```diff
@@ -29,6 +29,9 @@ class CRM_Upgrade_Incremental_php_SixEight extends CRM_Upgrade_Incremental_Base
    */
   public function upgrade_6_8_alpha1($rev): void {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+
+    $this->addTask('Add index to civicrm_translation_source', 'addIndex', 'civicrm_translation_source', 'source_key');
```

Comment:
@colemanw should this also be in a beta1 upgrade step for anyone that has updated to the RC already?